Source/ARTLocalDevice.m

```diff
@@ -70,7 +70,7 @@ + (ARTLocalDevice *)load:(NSString *)clientId storage:(id<ARTDeviceStorage>)stor
     ARTDeviceIdentityTokenDetails *identityTokenDetails = [ARTDeviceIdentityTokenDetails unarchive:identityTokenDetailsInfo];
     device->_identityTokenDetails = identityTokenDetails;
 
-    [device setDeviceToken:[storage objectForKey:ARTDeviceTokenKey]];
+    [device setDeviceToken:[storage objectForKey:ARTAPNSDeviceTokenKey]];
```

Comment:
This method should be called `setAPNSDeviceToken`

Reply:
done f9c88fd